README.md

```diff
@@ -48,6 +48,15 @@ Resources:
 
 ### Installation
 
+#### _Important info_
+
+If you were using this project in the release v0.1 and before, you need to run the dynamodb key migration script 
```

Comment:
... To run this script you will need to have at least a read access to CloudFormation and a read/write access to the DynamoDB table.